### DISCUSSION TITLE
Cannot send verification email over port 25

### DISCUSSION THREAD

**Discussion by doc-jay** [anonaddy/docker#261] [General]
I've just installed the 'latest' tag and I've setup my account.  I cannot receive my verification email because postfix is sending it over port 25 to my hosted provider, which has that port closed.  In my addy.env file, I've modified 'POSTFIX_SMTP_TLS=true', but it still tries to send the verification email over port 25.

How can I have postfix use TLS instead?

thank you

**Comment by fir3drag0n** [comment]
is there a solution to change the outgoing port, such as 587?
